azure-mgmt-keyvault/src/main/java/com/microsoft/azure/management/keyvault/implementation/SecretImpl.java

```diff
@@ -97,7 +97,16 @@ public boolean managed() {
 
                     @Override
                     protected ServiceFuture<SecretBundle> callAsync() {
-                        return vault.client().getSecretAsync(secretItem.identifier().identifier(), null);
+                        if (secretItem.attributes().enabled()) {
+                            return vault.client().getSecretAsync(secretItem.identifier().identifier(), null);
+                        } else {
+                            SecretBundle secretBundle = new SecretBundle()
+                                    .withId(secretItem.id())
+                                    .withAttributes(secretItem.attributes())
+                                    .withContentType(secretItem.contentType())
+                                    .withTags(secretItem.tags());
+                            return ServiceFuture.fromBody(Observable.just(secretBundle), null);
```

Comment:
What would be the difference if we create SecretBundle this way?

Reply:
The property secretBundle.value() will be NULL. For the enabled secrets, value will be returned by server side.

Reply:
That probably is acceptable. From our API user can know it is disabled, correct?

Reply:
I update the comments at Secret interface. Meanwhile, the status of enabled/disabled can be found via `secret.attributes().enabled()`.